lms/templates/courseware/courses.html

```diff
@@ -45,9 +57,19 @@
   </header>
 
   <section class="courses-container">
+
+    <div id="discovery-search">
+      <form>
+        <input placeholder="${_('Search for a course')}" type="text"/>
+        <button type="submit" class="button postfix" aria-label="${_('Search')}">
+          <i class="icon fa fa-search" aria-hidden="true"></i>
+        </button>
+      </form>
+    </div>
+
     <section class="courses">
       <ul class="courses-listing">
-	%for course in courses:
+	      %for course in courses:
```

Comment:
wassup with this? Spacing seems very odd here - this file seems to have mixed spaces / tabs; Also, is often using 2 spaces - despite that being my favourite, it is not our standard.